doc/design/couler_sqlflow.md

```diff
@@ -20,4 +20,51 @@ The migration includes the following parts:
 
 1. Instead of having multiple codegens, let us have only one, `codegen_couler.go`, which translates the [Intermediate Representation](/doc/design/intermediate_representation.md)(IR) of a SQL program into a Couler program. Then, SQLFlow can run the Couler compiler to convert further and execute the workflow.
 
-For example, `codegen_couler.go` converts a `SELECT ... TO TRAIN` statement into the call to `sqlflow.couler.{xgboost,tensorflow.elasticdl}.train(train_ir)`, which `train_ir` is the SQLFlow IR with JSON format.
+For example, `codegen_couler.go` converts a `SELECT ... TO TRAIN` statement into the call to `sqlflow.couler.{xgboost,tensorflow.elasticdl}.train(...)`.
+
+## Couler Step Function
+
+For example, an extended SQL `SELECT ... TO TRAIN xgboost.booster ...` would be compiled into a
+an Couler step function by `codegen_couler.go`:
+
+``` python
```

Comment:
Where can we specify configurations like datasource and session?

Reply:
We can pass them as the environment variable of `couler.run_container(env={"SQLFLOW_DATASOURCE":"mysql://(127.0.0.1:3306)..."}...)`